src/include/sof/lib/dai.h

```diff
@@ -228,6 +228,22 @@ static inline int dai_get_info(struct dai *dai, int info)
 	return ret;
 }
 
+static inline void dai_write(struct dai *dai, uint32_t reg, uint32_t value)
+{
+	io_reg_write(dai_base(dai) + reg, value);
```

Comment:
#include <sof/lib/io.h>

Reply:
@tlauda noticed this and fixed. The build even fails for IMX :)